applications/FluidDynamicsApplication/custom_conditions/navier_stokes_wall_condition.h

```diff
@@ -333,9 +335,15 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) NavierStokesWallCondition : public
 
     void CalculateNormal(array_1d<double,3>& An);
 
-    void ComputeGaussPointLHSContribution(BoundedMatrix<double,TNumNodes*(TDim+1),TNumNodes*(TDim+1)>& lhs, const ConditionDataStruct& data);
+    void ComputeGaussPointLHSContribution(
+        BoundedMatrix<double,TNumNodes*(TDim+1),TNumNodes*(TDim+1)>& rLHS,
+        const ConditionDataStruct& rData,
+        const ProcessInfo& rProcessInfo);
 
-    void ComputeGaussPointRHSContribution(array_1d<double,TNumNodes*(TDim+1)>& rhs, const ConditionDataStruct& data);
+    void ComputeGaussPointRHSContribution(
+        array_1d<double,TNumNodes*(TDim+1)>& rRHS,
```

Comment:
```suggestion
        array_1d<double, LocalSize>& rRHS,
```
Wouldn't it be better to use `BoundedVector` for consistent naming? Would that be lot of implications in lots of places?

Reply:
Agree. I'd suggest to change all of these in a single PR.